### ISSUE TITLE
No pause between rentals, and my rig immediately got rented at a low price

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#1190]
Please can you check my settings? I want an hour pause between rentals to recalibrate pricing.

 "MiningRigRentals": {
    "EnableMining": "0",
    "EnableAutoCreate": "1",
    "EnableAutoPrice": "1",
    "EnableMinimumPrice": "1",
    "PriceBTC": "0",
    "PriceFactor": "1.4",
    "PriceCurrencies": "BTC,LTC,ETH,BCH",
    "Title": "David Brown | %algorithm% | %workername% | BTC | ETH | LTC | BCH",
    "Description": "Please allow up to 3 minutes for the rental to begin.",
    "Worker": "$WorkerName",
    "Penalty": 0,
    "Algorithm": "",
    "ExcludeAlgorithm": "",
    "CoinName": "",
    "ExcludeCoin": "",
    "CoinSymbol": "",
    "ExcludeCoinSymbol": "",
    "MinerName": "",
    "ExcludeMinerName": "",
    "FocusWallet": "",
    "AllowZero": "1",
    "EnableAutoCoin": "0",
    "EnablePostBlockMining": "0",
    "CoinSymbolPBM": "",
    "DataWindow": "",
    "StatAverage": "",
    "MaxMarginOfError": "100",
    "SwitchingHysteresis": "",
    "User": "samdave",
    "API_Key": "f332ceea8e90d1019235d72abb9c1f9bcc8473f3b73f8e76c84899e53b54792f",
    "API_Secret": "8f38e49c251ed624c960c1721338b9421da010fa8769bbcf97d36c5747852a0d",
    "AutoCreateMinProfitPercent": "0",
    "AutoCreateMinCPUProfitBTC": "0.00001",
    "AutoCreateMaxMinHours": "24",
    "AutoCreateAlgorithm": "",
    "EnableAutoUpdate": "1",
    "AutoPriceModifierPercent": "0",
    "MinHours": "6",
    "MaxHours": "96",
    "EnableUpdateTitle": "1",
    "AutoUpdateMinPriceChangePercent": "1",
    "EnablePowerDrawAddOnly": "0",
    "PowerDrawFactor": "1.0",
    "UseWorkerName": "",
    "ExcludeWorkerName": "",
    "EnableUpdateDescription": "1",
    "EnableUpdatePriceModifier": "0",
    "EnableAutoExtend": "0",
    "AutoExtendTargetPercent": "95",
    "AutoExtendMaximumPercent": "5",
    "AutoBonusExtendForHours": "0",
    "AutoBonusExtendByHours": "0",
    "ProfitAverageTime": "Hour",
    "PauseBetweenRentals": "1h",
    "EnableAutoBenchmark": "1",
    "MaxAllowedLuck": "",
    "MaxTimeSinceLastBlock": ""

**Comment by samdave123** [comment]
Could the pause have been broken by RBM updating mid-rental?

**Comment by samdave123** [comment]
Maybe a bot got in there too fast?

**Comment by acos0874** [comment]
Hide your API details

**Comment by samdave123** [comment]
> 
> 
> Hide your API details

They aren't my real API details, but thanks for your concern.

**Comment by RainbowMiner** [comment]
Hi, I think, somebody snatched your rig just in between the end of the rental and the start of the next RainbowMiner round.

**Comment by samdave123** [comment]
> 
> 
> Hi, I think, somebody snatched your rig just in between the end of the rental and the start of the next RainbowMiner round.

Any way this can be easily fixed/mitigated for the future? I've got a very long low-paying rental now.

**Comment by RainbowMiner** [comment]
> Any way this can be easily fixed/mitigated for the future? I've got a very long low-paying rental now.

I am currently reviewing price updates while a rental is on. That should solve the issue, I hope.

**Comment by samdave123** [comment]
> 
> 
> > Any way this can be easily fixed/mitigated for the future? I've got a very long low-paying rental now.
> 
> I am currently reviewing price updates while a rental is on. That should solve the issue, I hope.

That would be great. I'm guessing it would also remove the requirement to have a pause between rentals to update the pricing, so I can make my rig available immediately after a rental ends and get good pricing?

**Comment by RainbowMiner** [comment]
Yes, there is, however, one thing to weigh in. If the rig updates in between (and there are miner updates), it will go into state "Benchmarking", in the background. During that state, the profit files do not update, because the list of the active miners is then incomplete. So we will have to make pretty sure, that the rig doesn't update itself during a rental (this might eventually happen, if you have a bad rental, where the miners switch back to normal operation for periods of time)
